### PR TITLE
[Term Entry] Create address.md

### LINKED ISSUE #2641: [Entry] Subject: <address> tag
### Reason/inspiration (optional)

I recently learned about the `<address>` element through a blog post I recently published, and I think it would be a good idea to add it to the [Semantic HTML entry](https://www.codecademy.com/resources/docs/html/semantic-html) so that learners know a new way to implement contact information in the footer of their projects.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by CBID2**
### Description

I added information about the `<address>` tag to the semantic HTML tag entry. This would help learners learn how to implement contact information effectively. 

- Closes #2641 
### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by CBID2** [comment]
@SSwiniarski, this PR is ready for review.

File: content/html/concepts/semantic-html/terms/address/address.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`<address>`** tag is a semantic HTML element that is used to indicate that an organization or person's contact information is being provided. This tag can include information such as a mailing address, links to social media, email, phone number, URLs, etc. This element can also be used in various areas such as the top of the webpage or its bottom.
+
+**Note:** Only contact information should be presented within the `<address>` tag. Content such as an blog post's publication date should be use within their respective elements, for example, the `<time>` tag.
+
+## Syntax
+
+```html
```
Comment (dakshdeepHERE) [inline content/html/concepts/semantic-html/terms/address/address.md:21]:
```suggestion
```pseudo 
```

File: content/html/concepts/semantic-html/terms/address/address.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+<address>
+  Written by <a href="mailto:jasminebakes@gmail.com">Jasmine Sanchez</a>.<br />
+  Visit us at:<br />
+  Fairytale Confections<br />
+  Box 354, Agrabah<br />
+  USA
+</address>
+```
```
Comment (dakshdeepHERE) [inline content/html/concepts/semantic-html/terms/address/address.md:51]: Why don't we change this example into codebyte?

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
A comment and a suggestion

File: content/html/concepts/semantic-html/terms/address/address.md (hunk 2 above)
Comment (CBID2) [reply content/html/concepts/semantic-html/terms/address/address.md:51]: I think example is best

Comment (SSwiniarski) [reply content/html/concepts/semantic-html/terms/address/address.md:51]: Not to mention there's no HTML implemented in codebytes.

File: content/html/concepts/semantic-html/terms/address/address.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+---
+Title: '<address>'
+Description: 'Represents the contact information of a business or person '
```
Comment (SSwiniarski) [inline content/html/concepts/semantic-html/terms/address/address.md:3]:
```suggestion
Description: 'Represents the contact information of a business or person.'
```

File: content/html/concepts/semantic-html/terms/address/address.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  - 'Semantic'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`<address>`** tag is a semantic HTML element that is used to indicate that an organization or person's contact information is being provided. This tag can include information such as a mailing address, links to social media, email, phone number, URLs, etc. This element can also be used in various areas such as the top of the webpage or its bottom.
+
+**Note:** Only contact information should be presented within the `<address>` tag. Content such as an blog post's publication date should be use within their respective elements, for example, the `<time>` tag.
```
Comment (SSwiniarski) [inline content/html/concepts/semantic-html/terms/address/address.md:17]:
```suggestion
> **Note:** Only contact information should be presented within the `<address>` tag. Content such as an blog post's publication date should be use within their respective elements, for example, the `<time>` tag.
```

File: content/html/concepts/semantic-html/terms/address/address.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+## Syntax
+
+```html 
+<address>
+  <!-- contact information goes here -->
+</address>
+```
+
+## Example 1
```
Comment (SSwiniarski) [inline content/html/concepts/semantic-html/terms/address/address.md:27]: I think we should use a fictitious domain for the e-mail so we don't inadvertently publish someone's real email.
Also with the phone number, use 316 555 2398 so it doesn't connect with someone's real number.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@CBID2 I did a first review and left a few comments.

File: content/html/concepts/semantic-html/terms/address/address.md (hunk 2 above)
Comment (dakshdeepHERE) [reply content/html/concepts/semantic-html/terms/address/address.md:51]: > Not to mention there's no HTML implemented in codebytes.

what are the lang supported on codebyte ?

**Comment by CBID2** [comment]
> @CBID2 I did a first review and left a few comments.

Thanks @SSwiniarski! 😊 If you're free, can you do a review on [this](https://github.com/Codecademy/ugc/pull/56)? I'm concerned about my PR being stale.

File: content/html/concepts/semantic-html/terms/address/address.md (hunk 5 above)
Comment (CBID2) [reply content/html/concepts/semantic-html/terms/address/address.md:27]: Good idea @SSwiniarski! 😊

**Comment by CBID2** [comment]
> @CBID2 I did a first review and left a few comments.

Made the changes @SSwiniarski

File: content/html/concepts/semantic-html/terms/address/address.md (hunk 2 above)
Comment (SSwiniarski) [reply content/html/concepts/semantic-html/terms/address/address.md]: C++, C#, Go, JavaScript, PHP, Python, Ruby, Scheme

**Comment by SSwiniarski** [comment]
> > @CBID2 I did a first review and left a few comments.
> 
> Made the changes @SSwiniarski

@CBID2 You still need to change the "gmail.com" in the email addresses to something fictitious.

**Comment by CBID2** [comment]
> > > @CBID2 I did a first review and left a few comments.
> 
> > 
> 
> > Made the changes @SSwiniarski
> 
> 
> 
> @CBID2 You still need to change the "gmail.com" in the email addresses to something fictitious.

Done @SSwiniarski

**Review by SSwiniarski — APPROVED** [review]
@CBID2 Looks good for a second review.

**Comment by CBID2** [comment]
> @CBID2 Looks good for a second review.

Great @SSwiniarski! Can you do a review on [this](https://github.com/Codecademy/ugc/pull/56) please?

**Review by caupolicandiaz — APPROVED** [review]
@CBID2 @SSwiniarski I just made a few small edits. Looks good to go

**Comment by SSwiniarski** [comment]
@CBID2 Congrats on the new entry. Live link here: https://www.codecademy.com/resources/docs/html/semantic-html/address

image

**Pull request merged**